MetaMask/metamask-extension
# fix: PPOM - Report False Positives - missing Chain Name

## **Description**

Whenever we trigger a signature/transaction, if we click on the Report False Positive link we see that the Chain Name is left empty. 
Affected flows:
- in Mainnet, only Singatures
- in the rest of nets, Singatures + Transactions

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23195?quickstart=1)

## **Related issues**

Fixes: #23067 

## **Manual testing steps**

1. Enable Blockaid
2. Select Mainnet
3. Trigger a malicious signature
4. Click Report False Positive
5. See details
6. See Chain Name missing
7. Checkout this branch and deploy
8. Repease 1-5
9. See Chain name is now displayed

## **Screenshots/Recordings**

### **Before**

https://github.com/MetaMask/metamask-extension/assets/54408225/d9a08fb6-e2ce-49c7-9812-76a0742e48fa

### **After**

https://github.com/MetaMask/metamask-extension/assets/44811/b5d7c4b6-627d-49d4-9817-545cfd3c65b2

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
